frontend/src/app/header.tsx

```diff
@@ -32,7 +35,17 @@ const VoucherNotification: FC = () => {
 
 export const Header: FC = () => {
     return (
-        <Group h="100%" px={20}>
+        <Stack
+        align="stretch"
+        justify="flex-start"
+        gap="xs"
+        >
+            <Group bg="gray">
+                <Center w="100%">
+                <Anchor href="/notification" underline="never">We are in Alpha stage, read more.</Anchor>
```

Comment:
```suggestion
                <Anchor href="/notification" underline="never">We are in alpha stage. Click here to learn more.</Anchor>
```

Reply:
I would like to keep it short and also would like to avoid repeating "learn more".
What do you think about "We are in alpha stage, click here for more info"? 🤔

Reply:
Sounds good to me. :-)